cl/recap/mergers.py

```diff
@@ -601,13 +601,15 @@ def add_docket_entries(d, docket_entries, tags=None):
     :param docket_entries: A list of dicts containing docket entry data.
     :param tags: A list of tag objects to apply to the recap documents and
     docket entries created or updated in this function.
-    :returns tuple of a list of RECAPDocument objects created and whether the
-    any docket entry was created.
+    :returns tuple of a list of RECAPDocument objects created, whether
+    any docket entry was created and a list of created or existing
+    RECAPDocument objects.
```

Comment:
```suggestion
    DocketEntry objects.
```

Also, should this return be reordered to be hierarchical: `des_created, rds_created, content_updated`?

Reply:
Yeah, I've changed the order, updated the docstrings accordingly, and updated the code where this return is used across other methods.